src/index.js

```diff
@@ -99,7 +99,8 @@ app.use(session({
     secret: config.sessionSecret,
     store: sessionStore,
     resave: true,
-    saveUninitialized: false
+    saveUninitialized: false,
+    cookie: {maxAge: 900000}
```

Comment:
Another minor comment: trailing comma is appropriate here.

Reply:
> It looks like if the limit is reached all other sessions are logged out. This does not seem to be a reasonable behavior. Otherwise this looks good.

All other sessions for that _user_ are logged out yes, which is intended behavior.

Reply:
The whole point to it is to restrict users to x amount of logged in devices at once.